src/docs/Components/Tabs/previews/Tabs-all-stats.preview.html

```diff
@@ -0,0 +1,27 @@
+<div class="example">
```

Comment:
As far as the preview file names are concerned, it would be best if everything is in lower case.